lib/dor/update_marc_record_service.rb

```diff
@@ -172,7 +172,7 @@ def get_x2_constituent_info
 
     def released_to_Searchworks
       rel = @druid_obj.released_for
-      rel['Searchworks']['release']
+      rel.blank? || rel['Searchworks'].blank? || rel['Searchworks']['release'].blank? ? false : rel['Searchworks']['release']
```

Comment:
I'm curious about the string `Searchworks` here. Where is that coming from?

Currently in prod we have 3 or 4 variations of this, and I setup Argo to do something different. Can we figure out what this should be or have it configurable?